web_src/js/index.js

```diff
@@ -242,6 +242,8 @@ function initReactionSelector(parent) {
         let react = content.find('.segment.reactions');
         if (!resp.empty && react.length > 0) {
```

Comment:
This change should do the same, this way no new else if is needed
```suggestion
        if ((!resp.empty || resp.html === '') && react.length > 0) {
```